src/ehrdata/dt/datasets.py

```diff
@@ -81,7 +81,10 @@ def mimic_iv_omop(backend_handle: DuckDBPyConnection, data_path: Path | None = N
             print(f"Failed to download the file. Status code: {response.status_code}")
             return
 
-    return _set_up_duckdb(data_path + "/1_omop_data_csv", backend_handle)
+    extracted_folder = next(
+        (folder for folder in data_path.iterdir() if folder.is_dir() and "_csv" in folder.name), data_path
+    )
+    return _set_up_duckdb(extracted_folder, backend_handle)
 
 
 def gibleed_omop(backend_handle: DuckDBPyConnection, data_path: Path | None = None) -> None:
```

Comment:
Adding more information to the docstrings would be great; see e.g. other ehrapy datasets that we already have, with links to where the data is from, and that this is in the omop data format with maybe a link to the omop data description